### PR TITLE
Consolidate gradle.properties files

### PR DESCRIPTION
**Pull request by kwonye**
We currently have two `gradle.properties-example` files, and both of them address two things: performance and property values for the project.

I propose we consolidate both to the one in the root directory for multiple reasons:

1. There is no need for two separate files
2. The one in `./WordPress/` doesn't show up in Android Studio when in Android view. Plus it's a project file, not a module file.
3. It's almost a requirement that the developer modifies the jvm args and this will make that setting easier to find.

### CODE REVIEW

File: WordPress/build.gradle [hunk 1]
```diff
@@ -156,8 +156,8 @@ configurations.all {
 }
 
 task generateCrashlyticsConfig(group: "generate", description: "Generate Crashlytics config") {
-    def outputFile = new File("${rootDir}/WordPress/crashlytics.properties")
-    def inputFile = file("${rootDir}/WordPress/gradle.properties")
+    def outputFile = new File("${rootDir}/crashlytics.properties")
```
Comment (maxme) [inline WordPress/build.gradle]: This won't work Crashlytics needs his properties in the app root (ie `WordPress/`)

**Comment by maxme** [comment]
Travis config must be updated

**Comment by maxme** [comment]
> It's almost a requirement that the developer modifies the jvm args and this will make that setting easier to find.

Yes, so maybe we should just rename `/gradle.properties-example` to `/gradle.properties` and set some defaults value (instead of commented values).

IMO that change is a bit risky and doesn't add much value, I'm also not sure about the implication of moving this ids to the root module (libs can now read them).

**Comment by maxme** [comment]
Note: The release check script must be updated if have to apply that change.

**Comment by kwonye** [comment]
>Yes, so maybe we should just rename /gradle.properties-example to /gradle.properties and set some defaults value (instead of commented values).

We actually had that at first but changed it to `.properties-example` because people didn't want to see their changes in their working copy all the time.

> IMO that change is a bit risky and doesn't add much value, I'm also not sure about the implication of moving this ids to the root module (libs can now read them).

Hmm, I don't see how this change is risky, can you explain please? The way I see it this will bring our project more in-line with the expected project structure of an Android project.

> Note: The release check script must be updated if have to apply that change.

Updated, I'm sorry I missed those! 😕

**Comment by kwonye** [comment]
@maxme ready for another pass 👍

**Comment by maxme** [comment]
I squashed the commits into one so we can revert it quickly if something wrong happen with the next release.

⚠️ @mzorz make sure to copy your gradle.properties file to the root directory, this will only be used for 7.5 release, so be careful with 7.4 ⚠️

**Pull request merged**